### DISCUSSION TITLE
Ability To Move Solve Times Between Seasions

### DISCUSSION THREAD

**Discussion by Wardeneo** [CubeLabsNZ/CubeTime#38] [Ideas]
Firstly I’d like to add todays update with the scramble graphic and screen timeout block during solves are great additions! 👍🏾

Whilst using the app today I did a couple of solves and realised I was in wrong session (I have multiple sessions for different cubes). I’d like to be able to move a solve from 1 session to another, not just the ability to delete solves

cheers Wardeneo

**Comment by Vic55105** [comment]
That a good Idea

**Comment by PistonUser** [comment]
Maybe even make a selection system to move solves and stuff

**Comment by xbjfk** [comment]
Coming in v2, including multiselect ability (releasing soon for real 😭)

**Comment by pdtxie** [comment]
v2 releasing today!

**Comment by xbjfk** [comment]
apple has accepted v2!
